Jantimizer/Tools/Regex/RegexHelperFunctions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Tools.Regex
+{
+    public static class RegexHelperFunctions
+    {
+
+        public static T GetRegexVal<T>(Match regexMatch, string name)

```

Comment:
This is a rather weird function. Would benefit from being split into two, one being `GetRegexVal` and one `GetRegexValOrNull` instead

Reply:
As a sidenote, it would maybe be a good idea to restrict `T` to be `where T : IConvertible`

Reply:
This file would maybe also benefit from being in the `Helpers` folder instead